tcf_website/templates/search/search.html

```diff
@@ -16,113 +16,140 @@ <h2>
 
       <div class="search-results container p-5">
 
+        <!-- Tabs for displaying result categories: Courses, Departments, Instructors -->
         <div id="result-nav" class="result-nav row mb-5">
           <div class="col-12">
-            <div class="btn-group btn-group-sm" role="group" aria-label="Basic example">
+            <div class="btn-group btn-group-sm" role="toolbar">
               <a class="btn bg-tcf-indigo text-white" disabled>See results for</a>
-              <a href="#departments" type="button" class="btn btn-light">Departments</a>
-              <a href="#courses" type="button" class="btn btn-light">Courses</a>
-              <a href="#instructors" type="button" class="btn btn-light">Instructors</a>
+              <a href="#courses" id="course-tab" class="btn btn-light collapsed" autofocus
```

Comment:
small nitpick but it might be better to use `active` instead of `focus` on the current tab so that it persists after you click away

Reply:
I tried it out, but it's difficult to toggle active state when you click between tabs since I have `data-toggle="collapse"` instead of `data-toggle="button"` (unless we want to write more custom JQuery for it which isn't great for thin views)